### DISCUSSION TITLE
Support for legacy OPC

### DISCUSSION THREAD

**Discussion by duduyoyo** [pyscada/PyScada#143] [General]
Is it possible to support legacy OPC like DA/AE/HDA? It was widely adopted and available in every vendor device. Thanks!

**Comment by clavay** [comment]
I don't have any access to a device using OPC classic (DA/AE/HDA).
It should be possible using a library as  https://github.com/j3mg/openopc or https://github.com/iterativ/openopc2

First you should try to use one of this library to communicate with you device.

After having a script working to read and/or write to the device, I can help you to create a PyScada plugin to integrate the protocol into the PyScada architecture.
